xsd/netex_part_1/part1_networkDescription/netex_line_version.xsd

```diff
@@ -490,19 +489,19 @@ Rail transport, Roads and Road transport
 			</xsd:element>
 			<xsd:element ref="TypeOfProductCategoryRef" minOccurs="0"/>
 			<xsd:element ref="TypeOfServiceRef" minOccurs="0"/>
-			<xsd:element name="UserTypes" type="UserTypeListOfEnumerations" minOccurs="0">
+			<xsd:element name="AccessModes" type="AccessModeListOfEnumerations" minOccurs="0">
```

Comment:
1. Given that the constraint describes a case that suggest 'if not limited, do usual'. Shouldn't we call it something like AccessModeLimitation?
2. I would also expect that more options should be able to be provided. Is it my correct interpretation that this is a single value from the enumeration, because it is not a list?

Reply:
@nick-knowles  @Aurige

Reply:
It isn't always a limitation / restriction: for instance, *car* or *motorcycle* as access mode is an extension of the typical case.